Demos/SwiftUI Demo/SwiftUI Demo/ContentView.swift

```diff
@@ -18,7 +18,7 @@ struct ContentView: View {
 
     init() {
         guard let config = try? LogtoConfig(
-            endpoint: "https://logto.dev",
+            endpoint: "http://localhost:3001",
```

Comment:
Please revert this change as it's for local test

Reply:
sure. dkw logto.dev still have the resource issue. will figure out tmw.

Reply:
👌